README.md

```diff
@@ -100,40 +100,36 @@ https://arewecompressedyet.com/analyzer/?d=https://people.xiph.org/~mbebenita/an
 
 ## Coding style
 Check code formatting with [rustfmt](https://github.com/rust-lang-nursery/rustfmt) before submitting a PR.
-rav1e currently uses the nightly version of rustfmt.
+rav1e currently uses the stable version of rustfmt.
 
-To install nightly:
+To install stable tool chain:
 
 ```
-rustup install nightly
+rustup install stable
 ```
 
-To install the nightly version of rustfmt:
+To install the rustfmt:
 
 ```
-rustup component add rustfmt-preview --toolchain nightly
+rustup component add rustfmt
 ```
 
 then
 
 ```
-cargo +nightly fmt -- --check
+cargo fmt -- --check
 ```
 
 You should also try [clippy](https://github.com/rust-lang-nursery/rust-clippy).
-Rust also uses nightly for clippy.
 
 To install clippy:
 
 ```
-rustup component add clippy-preview --toolchain nightly
+rustup component add clippy
 ```
 
-then
+then you can refer travis script for detailed cargo clippy [command](https://github.com/xiph/rav1e/blob/master/.travis.yml)
```

Comment:
The meaning of "travis script" may not be very clear for people who don't know what it is. The exact file name (clickable, as a link) would be more understandable.

I also feel like something important is missing from this sentence. I mean it's not immediately clear what I should look for and why. Could it be somehow addressed as well?